script/release_utils.sh

```diff
@@ -1,12 +1,16 @@
 #!/bin/bash
 set -e
 
+function getLatestTag {
+  git fetch --tags
+  git describe --tags $(git rev-list --tags --max-count=1)
+}
+
 function commit_list {
   local tag=${1:?}
   local repo_domain=${2:?}
   local repo_name=${3:?}
-  git fetch --tags
-  local previous_tag=`git describe --tags $(git rev-list --tags --max-count=1)`
+  local previous_tag=$(getLatestTag)
```

Comment:
why is getLatestTag the previous tag?

Reply:
mm, you are right. I thought that when the current commit is a tag this would return the previous one but I just checked in the latest release of circleCI and it's returning the current tag (so the comparison is empty). Did this work when you did the release of v1.0.0-alpha.6? I see you discarded the draft and you created the release notes on your own, did the draft include the list of commits of the release?

Reply:
it seems it doesn't work. I'll send a PR for fixing that.

Reply:
It didn't, I forgot to create an issue for it, sorry - thanks!